allow specifying `HEAD` as project dependency version for workspace (#1241)
finos/legend-studio: .changeset/cool-zoos-repair.md

---
'@finos/legend-studio': minor
---

> Update on the project dependency configuration to contain HEAD as a version option and added restriction on the creation of review to ensure no project dependency is set to the HEAD option.

Reviewer: ```suggestion
Allow the project dependency configuration to contain `HEAD` as a alid version option and added restrictions on review-creation to ensure no project dependency is set to the `HEAD` option in the project.
```
